buildSrc/src/main/java/UpdateChangelogTask.kt

```diff
@@ -0,0 +1,42 @@
+import kotlinx.serialization.json.Json
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import org.intellij.markdown.flavours.commonmark.CommonMarkFlavourDescriptor
+import org.intellij.markdown.html.HtmlGenerator
+import org.intellij.markdown.parser.MarkdownParser
+import java.io.File
+
+/** Convert a markdown changelog to HTML. */
+open class UpdateChangelogTask : DefaultTask() {
+
+    @get:Input lateinit var sourceFile: String
+    @get:Input lateinit var targetFile: String
+
+    private val markdownFlavour = CommonMarkFlavourDescriptor()
+    private val json = Json { prettyPrint = true }
```

Comment:
unused
```suggestion

```

Reply:
Good catch! Removed in [`51d0b31`](https://github.com/streetcomplete/StreetComplete/pull/5439/commits/51d0b312d1b6e6b02a877b3dc8a81a33d7708f75).